esp-hal/src/clock/mod.rs

```diff
@@ -43,6 +43,7 @@
 //! # Ok(())
 //! # }
 //! ```
+#![cfg_attr(not(feature = "rt"), allow(unused))]
```

Comment:
Not overly fond of this, but the clock_ll and soc clock modules are a bit messy at the moment quickly.